mux.go

```diff
@@ -363,9 +363,9 @@ func (r *Router) Walk(walkFn WalkFunc) error {
 	return r.walk(walkFn, []*Route{})
 }
 
-// SkipRouter is used as a return value from WalkFuncs to indicate that the
+// ErrSkipRouter is used as a return value from WalkFuncs to indicate that the
 // router that walk is about to descend down to should be skipped.
-var SkipRouter = errors.New("skip this router")
+var ErrSkipRouter = errors.New("skip this router")
```

Comment:
You can't rename `SkipRouter` to `ErrSkipRouter` without breaking existing user code. This change is not backwards compatible. You should revert this rename.